doc/book/validators/callback.md

```diff
@@ -123,6 +123,26 @@ if ($valid->isValid($input)) {
 }
 ```
 
+AS of PHP 5.5 you can use ::class resolution for given callback class.
+
+```php
+class MyClass
+{
+    public function __invoke($value)
+    {
+        // some validation
+        return true;
+    }
+}
+
+$valid = new Zend\Validator\Callback(MyClass::class);
```

Comment:
Nvm, was confused, then read the code for it. This breaks the principle of least possible astonishment for me.